docs/UserGuide.md

```diff
@@ -343,25 +407,28 @@ Format: `export [-f <file_path>]`
 * The file path can be either the absolute path or the relative path.
 * The relative path will be relative to the folder your jar file is located.
 * If the file path is not specified, it will be saved to `sourceControl.csv`.
-* The format of the csv file saved matches exactly the csv format used by the `import` command.
+* The format of the CSV file saved matches exactly the CSV format used by the `import` command.
 
 <div markdown="block" class="alert alert-primary">
 
 :bulb: **Tips:**<br>
 
-* The exported csv file can be imported again to restore the state of the application when the data was exported
+* The exported CSV file can be imported again to restore the state of the application when the data was exported
 
 </div>
 
 Examples:
-* `export -f student_data.csv` exports database as file `student_data.csv`.
-* `export -f /home/prof/CS1101S/student_data.csv` exports database to the given absolute path.
-* `export` exports database as file `sourceControl.csv`.
+* `export -f student_data.csv` 
+  * exports database as file `student_data.csv`.
```

Comment:
Maybe specify the file location where it will be exported to too? Same for `edit` below